DBMS/DB2/1.txt

```diff
@@ -0,0 +1,41 @@
+--ASSIGNMENT1
+-- Query to find the columns of table members
+SELECT * FROM members;
```

Comment:
here you use the below sql syntax:
Desc members.

Above query return the data store in the table.